lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -91,6 +92,9 @@ export class PublicClientApplication {
         // Initialize the browser storage class.
         this.browserStorage = new BrowserStorage(this.config.auth.clientId, this.config.cache);
 
+        // Initialize default authority instance
+        B2cAuthority.setKnownAuthorities(this.config.auth.knownAuthorities);
```

Comment:
Thoughts on moving this into `AuthorityFactory.createInstace()`? It's a pre-requisite for B2C to work, and right now we have to remember to do this every time we create an instance (that's not obvious from the code). Instead we can pass `this.config.auth.knownAuthorities` as a parameter and that way ensure that it's always called if set on the configuration.

Reply:
Chatted offline, doesn't make sense.